.storybook/preview.js

```diff
@@ -35,6 +35,30 @@ const cdsCoreAndShimStyles = [previewStyles, resetStyles, coreTokens, darkThemeC
 loadIcons();
 addDocs(docs);
 
+const customViewports = {
+  large: {
+    name: 'Large',
+    styles: {
+      width: '992px',
+      height: '992px',
+    },
+  },
+  medium: {
+    name: 'Medium',
+    styles: {
+      width: '768px',
+      height: '768px',
+    },
+  },
+  small: {
+    name: 'Small',
+    styles: {
+      width: '576px',
+      height: '576px',
+    },
+  },
+};
+
```

Comment:
`customViewports` will override standard ones. Can you add the standard as well?

Reply:
Added the INITIAL_VIEWPORTS from the addon-viewports for storybook

Reply:
Updated to have only the last ones plus the new ones after offline discussion

Reply:
https://github.com/vmware-clarity/ng-clarity/assets/132376042/1001638c-c005-4fe7-a4e7-945d87e38385